src/legacy/core_plugins/data/public/search/aggs/metrics/lib/parent_pipeline_agg_helper.ts

```diff
@@ -36,20 +36,6 @@ const metricAggFilter = [
   '!geo_centroid',
 ];
 
-const metricAggTitle = i18n.translate('data.search.aggs.metrics.metricAggTitle', {
-  defaultMessage: 'Metric agg',
-});
-
-const [metricAggSchema] = new Schemas([
-  {
-    group: 'none',
```

Comment:
same as with order agg, allowedAggs property on the agg param replaces the aggFilter on schema